docs/_data/berlin-2025-config.yaml

```diff
@@ -0,0 +1,224 @@
+    #   link: /tickets
+    # - text: See the Schedule
+    #   link: /schedule
+#    - text: See the Talks
+#      link: /speakers
+#    - text: Read the Guide
+#      link: /attendee-guide
+    # - text: Watch the Live Stream
+    #   link: /livestream
+    # - text: Conference recap
+    #   link: /news/thanks-recap
+    # - text: Browse the photos
+    #   link_absolute: https://www.flickr.com/photos/writethedocs/albums/72177720308088427
+    # - text: Watch the videos!
+    #   link_absolute: https://www.youtube.com/playlist?list=PLZAeFn6dfHpm4FboYSTD1Bs8Yp8k_JvAL
+
+tickets:
+  concierge:
+    price: €TBC
+  corporate:
+    price: €TBC
+  independent:
+    price: €TBC
+  student:
+    price: €TBC
+  virtual:
+    price: €TBC
+#  button: >
+#    .. raw:: html
+#
+#      <p style="margin: 2em 0;">
+#      <table border="0" cellpadding="0" cellspacing="0" style="background-color:#2ECC71; border:1px solid #4a4a4a; border-radius:5px;">
+#      <tr>
+#          <td align="center" valign="middle" style="color:#FFFFFF; font-family:Helvetica, Arial, sans-serif; font-size:16px; font-weight:bold; letter-spacing:-.5px; line-height:150%; padding-top:15px; padding-right:30px; padding-bottom:15px; padding-left:30px;">
+#            <a href="https://ti.to/writethedocs/write-the-docs-portland-2025/" target="_blank" style="color:#FFFFFF; text-decoration:none; border-bottom: none;">Get your ticket</a>
+#          </td>
+#      </tr>
+#      </table>
+#      </p>
+
+sponsorship:
+  second_draft:
+    price: €TBC
+  publisher:
+    price: €TBC
+  patron:
+    price: €TBC
+  keystone:
+    price: €TBC
+
+grants:
+#  url: https://docs.google.com/forms/d/e/1FAIpQLSf3fGfW_8W1x3oeKeoUpg8ZPLvyY4cQvO5Bii_OTrgW97cLNw/viewform?embedded=true
+#  ends: 'February 5, 2025'
+#  notification: 'February 14, 2025'
+#
+#shirts:
+#  url: https://shirt.writethedocs.org/
+#  ends: "May 31, 2025"
+
+date: # how do we handle these? human readable would be nice.
+  main: "**Oct 27-28, 2025**"
+  short: Oct 27-28, 2025
+  tickets_live: "June 2024"
+  month: October
+
+  # Conference is special for the index to show the combined conference days
+  conference:
+    event: Speaker Talks
+    date: October 27-28
+    summary: The main days of the conference. Listen to a diverse panel of speakers share their insights and experience.
+    icon: conference
+
+  # These are listed in the schedule
+#  day_one:
+#    event: Hike
+#    date: May 3
+#    # summary: We encourage you to go on a hike in your own beautiful home town.
+#    summary: If you're in town early, join us for the Hike. Portland has excellent access to the woods right in the city.
+#    icon: hike
+#    dotw: Saturday
+#  day_two:
+#    event: Writing Day
+#    date: May 4
+#    summary: Kick off the conference with Writing Day and work with fellow documentarians on a project. Followed by the opening reception.
+#    icon: writing
+#    dotw: Sunday
+#    writing_day_time: 9am-5pm
+#    reception_time: 5-7pm
+  day_one:
```

Comment:
Do we have a two page layout we can use here? I guess not.

Reply:
We do not have that.